projects/archiver/src/utils/backend-client.ts

```diff
@@ -73,15 +73,15 @@ export const getEditions = async (): Promise<Attempt<EditionsList>> => {
     const path = `${URL}editions`
     console.log(`Attempt to get editions list from path: ${path}`)
     const response = await fetch(path)
-    const maybeEditionsList = await attempt(response.json() as Promise<{
-        content: EditionsList
```

Comment:
this 'content' object was a result of not writing the getEdiitions endpoint correctly the first time - since fixed